packages/compiler-dom/package.json

```diff
@@ -5,8 +5,8 @@
   "main": "index.js",
   "module": "dist/compiler-dom.esm-bundler.js",
   "types": "dist/compiler-dom.d.ts",
-  "unpkg": "dist/compiler-dom/global.js",
-  "jsdelivr": "dist/compiler-dom/global.js",
+  "unpkg": "dist/compiler-dom/compiler-dom.global.js",
```

Comment:
It should be `dist/compiler-dom.global.js`

Fixed in fcb6c892

Reply:
Oh, thanks for spotting this.